### ISSUE TITLE
Dangling Symlinks in Debian Package

### ISSUE THREAD

**Issue by Johan2403** [GitCredentialManager/git-credential-manager#569]
**Which version of GCM are you using?**

gcm 2.0.605

**Which Git host provider are you trying to connect to?**

* [ ] Azure DevOps
* [ ] Azure DevOps Server (TFS/on-prem)
* [x] GitHub
* [ ] GitHub Enterprise
* [ ] Bitbucket
* [ ] Other - please describe

**Can you access the remote repository directly in the browser using the remote URL?**

From a terminal, run `git remote -v` to see your remote URL.

* [x] Yes
* [ ] No, I get a permission error
* [ ] No, for a different reason - please describe

---

**_[Azure DevOps only]_ What format is your remote URL?**

* [ ] Not applicable
* [ ] https://dev.azure.com/`{org}`/...
* [ ] https://`{org}`@dev.azure.com/`{org}`/...
* [ ] https://`{org}`.visualstudio.com/...

**_[Azure DevOps only]_ If the account picker shows more than one identity as you authenticate, check that you selected the same one that has access on the web.**

* [ ] Not applicable
* [ ] I only see one identity
* [ ] I checked each identity and none worked

---

**Expected behavior**

`/usr/local/bin/Atlassian.Bitbucket.UI` should point to `/usr/local/share/gcm-core/Release/Atlassian.Bitbucket.UI`
`/usr/local/bin/git-credential-manager-core` should point to `/usr/local/share/gcm-core/Release/git-credential-manager-core`
`/usr/local/bin/GitHub.UI` should point to `/usr/local/share/gcm-core/Release/GitHub.UI`

**Actual behavior**

`/usr/local/bin/Atlassian.Bitbucket.UI` points to `/usr/local/share/gcm-core/Atlassian.Bitbucket.UI`
`/usr/local/bin/git-credential-manager-core` points to `/usr/local/share/gcm-core/git-credential-manager-core`
`/usr/local/bin/GitHub.UI` points to `/usr/local/share/gcm-core/GitHub.UI`

**Logs**

Package installation:
```shell
$ sudo dpkg -i gcmcore-linux_amd64.2.0.605.12951.deb
Selecting previously unselected package gcmcore.
(Reading database ... 202401 files and directories currently installed.)
Preparing to unpack gcmcore-linux_amd64.2.0.605.12951.deb ...
Unpacking gcmcore (2.0.605.12951) ...
Setting up gcmcore (2.0.605.12951) ...
$ git-credential-manager-core configure
git-credential-manager-core: command not found
$ dpkg -L gcmcore
/.
/usr
/usr/local
/usr/local/bin
/usr/local/share
/usr/local/share/gcm-core
/usr/local/share/gcm-core/Release
/usr/local/share/gcm-core/Release/Atlassian.Bitbucket.UI
/usr/local/share/gcm-core/Release/GitHub.UI
/usr/local/share/gcm-core/Release/NOTICE
/usr/local/share/gcm-core/Release/git-credential-manager-core
/usr/local/share/gcm-core/Release/libHarfBuzzSharp.so
/usr/local/share/gcm-core/Release/libSkiaSharp.so
/usr/local/bin/Atlassian.Bitbucket.UI
/usr/local/bin/GitHub.UI
/usr/local/bin/git-credential-manager-core
$ ls -l /usr/local/bin
total 0
lrwxrwxrwx 1 1001 avahi-autoipd 40 Nov 16 21:15 Atlassian.Bitbucket.UI -> ../share/gcm-core/Atlassian.Bitbucket.UI
lrwxrwxrwx 1 1001 avahi-autoipd 45 Nov 16 21:15 git-credential-manager-core -> ../share/gcm-core/git-credential-manager-core
lrwxrwxrwx 1 1001 avahi-autoipd 27 Nov 16 21:15 GitHub.UI -> ../share/gcm-core/GitHub.UI
```

**Comment by ldennington** [comment]
Hi Johan - this has been fixed by #539. However, this has yet to be released. For now, the suggested workaround is to run:

`sudo mv /usr/local/share/gcm-core/Release/* /usr/local/share/gcm-core/ && sudo rm -rf /usr/local/share/gcm-core/Release/`

**Comment by Johan2403** [comment]
Hi Idennington,

Thanks for your reply. I've used the workaround and now it works fine.
I'll close this issue.
